jenkins/internal-ci/generic/nightly/nightly-performance-ci.groovy

```diff
@@ -184,7 +184,7 @@ pipeline {
                 manager.createSummary("${ICON}").appendText("<h3>Nightly CORTX Performance CI ${currentBuild.currentResult} </h3><p>Please check <a href=\"${BUILD_URL}/console\">Performance Sanity Execution logs</a> for more info <h4>Sanity Execution Logs:</h4>${clusterStatusHTML}", false, false, false, "red")
 
                 // Email Notification
-                if ( params.EMAIL_RECIPIENTS == "ALL" && currentBuild.result == "SUCCESS" ) {
+                if ( params.EMAIL_RECIPIENTS == "ALL" ) {
```

Comment:
why we removed build status check? is it not required?

Reply:
We have not removed the Build status check. We just enabled email notification to all the members even if the build failed. Earlier the email is not sent to the wider team if the build failed and `EMAIL_RECIPIENTS == "ALL"`